<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WahFormantShiftFrame</class>
 <widget class="QWidget" name="WahFormantShiftFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="smallDials">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>75</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page12">
    <widget class="QToasterDial" name="mixDial">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="unit">
      <string>%</string>
     </property>
    </widget>
    <widget class="QToasterEnumDial" name="pedalModeDial">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterEnumDial::Small</enum>
     </property>
     <property name="values">
      <stringlist>
       <string>Off</string>
       <string>Touch</string>
       <string>On</string>
       <string>Bypass@Stop</string>
       <string>Bypass@Heel</string>
       <string>Bypass@Toe</string>
      </stringlist>
     </property>
    </widget>
    <widget class="QToasterDial" name="duckingDial">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="minValue">
      <double>-5.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>5.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page22">
    <widget class="QToasterDial" name="touchBoostDial">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QToasterDial" name="touchReleaseDial">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QToasterDial" name="touchAttackDial">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToasterStompEditLCD" name="lcdDisplay">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>75</y>
     <width>420</width>
     <height>115</height>
    </rect>
   </property>
   <property name="color">
    <enum>QToasterLCD::Orange</enum>
   </property>
   <property name="currentPage">
    <enum>QToasterLCD::Page1</enum>
   </property>
   <property name="stompName">
    <string>Wah Wah</string>
   </property>
   <property name="value1Title">
    <string>Manual</string>
   </property>
   <property name="value2Title">
    <string/>
   </property>
   <property name="value3Title">
    <string>Pedal Range</string>
   </property>
   <property name="value4Title">
    <string>Pitch Shift</string>
   </property>
   <property name="value5Title">
    <string>Pedal Mode</string>
   </property>
   <property name="value6Title">
    <string>Mix</string>
   </property>
   <property name="value7Title">
    <string>Ducking</string>
   </property>
   <property name="value8Title">
    <string>Page</string>
   </property>
   <property name="value9Title">
    <string>Volume</string>
   </property>
   <property name="value10Title">
    <string/>
   </property>
   <property name="value11Title">
    <string/>
   </property>
   <property name="value12Title">
    <string/>
   </property>
   <property name="value13Title">
    <string>Touch Attack</string>
   </property>
   <property name="value14Title">
    <string>Touch Release</string>
   </property>
   <property name="value15Title">
    <string>Touch Boost</string>
   </property>
   <property name="value16Title">
    <string>Page</string>
   </property>
   <property name="scaleView">
    <enum>QToasterStompEditLCD::Empty</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="bigDials">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>560</width>
     <height>100</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page11">
    <widget class="QToasterDial" name="pitchShiftDial">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-36.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>36.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>1.000000000000000</double>
     </property>
     <property name="unit">
      <string/>
     </property>
     <property name="isActive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToasterDial" name="dummy2Dial">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Uni</enum>
     </property>
     <property name="minValue">
      <double>0.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="isActive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToasterDial" name="manualDial">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Uni</enum>
     </property>
     <property name="minValue">
      <double>0.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QToasterDial" name="pedalRangeDial">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-100.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>100.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>1.000000000000000</double>
     </property>
     <property name="unit">
      <string>%</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page21">
    <widget class="QToasterDial" name="dummy10Dial">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Uni</enum>
     </property>
     <property name="minValue">
      <double>0.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="isActive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToasterDial" name="volumeDial">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-5.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>5.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QToasterDial" name="dummy11Dial">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Uni</enum>
     </property>
     <property name="minValue">
      <double>0.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="isActive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToasterDial" name="dummy12Dial">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Uni</enum>
     </property>
     <property name="minValue">
      <double>0.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string/>
     </property>
     <property name="isActive">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToasterEnumDial" name="pageDial">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterEnumDial::Small</enum>
   </property>
   <property name="values">
    <stringlist>
     <string>One</string>
     <string>Two</string>
    </stringlist>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QToasterDial</class>
   <extends>QWidget</extends>
   <header>qtoasterdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterEnumDial</class>
   <extends>QWidget</extends>
   <header>qtoasterenumdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterStompEditLCD</class>
   <extends>QWidget</extends>
   <header>QToasterStompEditLCD.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>volumeDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue9(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>duckingDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue7(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue1(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue6(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bigDials</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smallDials</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue16(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue8(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pedalModeDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue5(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pedalRangeDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue3(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>touchAttackDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue13(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>touchBoostDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue15(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>touchReleaseDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue14(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pitchShiftDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue4(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
